Vagrantfile

```diff
@@ -112,11 +112,21 @@ Vagrant.configure(2) do |config|
         end
     end
 
-    if not setupConfig['subhosts'].empty?
+    if not setupConfig['subhosts'].empty? or not setupConfig['aliases'].empty?
         aliases = []
-        setupConfig['subhosts'].each do |subhost|
-            aliases.push(subhost['subhostname'] + '.' + setupConfig['hostname'])
+
+        if not setupConfig['subhosts'].empty?
+            setupConfig['subhosts'].each do |subhost|
+                aliases.push(subhost['subhostname'] + '.' + setupConfig['hostname'])
```

Comment:
no aliases for subhosts?

Reply:
I thought of it as you need either subhosts or aliases but not both. How would the final setup look like if you want aliases and subhosts?

Reply:
host
alias
subhost.host
subhost.alias